internal_transfers/actions/tests/utils.spec.ts

```diff
@@ -16,3 +18,56 @@ describe("transferInvolves(transfer, address)", () => {
     expect(transferInvolves(transfer, address3)).toBe(false);
   });
 });
+
+describe("getTxDataFromHash(hash)", () => {
+  test("does its job on basic transaction WETH unwrap", async () => {
+    const provider = ethers.getDefaultProvider(
+      process.env["NODE_URL"] || "NODE_URL"
```

Comment:
Discussion point: I've seen this in our Python projects as well, but in my opinion, our tests shouldn't be dependent on 3rd party services, and I believe this should be mocked and not relied upon.